packages/playground/src/weblets/profile_manager.vue

```diff
@@ -157,7 +157,7 @@
                               :disabled="creatingAccount || activatingAccount || activating"
                               @click:append="reloadValidation"
                             >
-                              <template v-slot:prepend-inner v-if="mnemonic.length > 0">
+                              <template v-slot:prepend-inner v-if="mnemonic && keypairType !== KeypairType.ed25519">
```

Comment:
why u are not supporting this keypair type ??

Reply:
If i have mnemonic of type ed25519 and activated, I can login.

Reply:
> why u are not supporting this keypair type ??

I see from the message that it's not supported

Reply:
It's not supported to create mnemonic or activate pre existing one, but u can login if u have one already.

Reply:
I have activated one of type ed25519 and i can connect with it.

Reply:
fixed